elife/php7.sls

```diff
@@ -119,3 +119,22 @@ composer-permissions:
             - group
         - require:
             - cmd: update-composer
+
+composer-global-stability-settings:
+    cmd.run:
+        - name: |
+            composer global config minimum-stability dev 
+            composer global config prefer-stable true 
```

Comment:
You mean making a release? Because here we are installing `elife/proofreader-php` as a global package, so the target for these flags is the global `composer.json` in `/home/elife/.composer`. Where else can they be specified?